app/views/static_pages/home.html.erb

```diff
@@ -1,8 +1,25 @@
 <%= provide :title, "Home" %>
-
 <h1>Sample App</h1>
 <p>
- This is the home page for the
- <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
- sample application.
-</p
+  This is the home page for the
+  <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+  sample application.
+</p>
+
+<% if logged_in? %>
+  <div class="row">
+    <aside class="col-md-4">
+      <section class="user_info">
+        <%= render "shared/user_info" %>
+      </section>
+      <section class="micropost_form">
+        <%= render "shared/micropost_form" %>
+      </section>
+    </aside>
+    <div class="col-md-8">
+      <h3><%= "Micropost Feed" %></h3>
+      <%= render "shared/feed" %>
+    </div>
+  </div>
+<% else %>
```

Comment:
indent please

Reply:
I am sorry but I still could not detect where is the indent error. I already compared with the slides and few coding of my friends, still I could not find the indent error.